site/src/components/code-block.tsx

```diff
@@ -152,6 +152,10 @@ const CodeBlock: React.FC<CodeBlockProps> = (props) => {
   if (code.length === 0) {
     return null;
   }
+
+  const codeWithDollarPrefixRemoved = code.startsWith('$ ')
+    ? code.substring(2)
```

Comment:
- Should we drop `$ ` only when `props.language` is `bash`?
- Should we consider multiple lines where multiple `$ ` prefixes are used? For example:
  ```bash
  # Clean up the Gradle build output
  $ ./gradles clean
  # Run Gradle tests
  $ ./gradlew test
  ```

Reply:
- The main purpose of this PR is to remove `$ ` when `props.language` is `bash`. So I'll filter out languages other than `bash` 
- Seems reasonable to me. I'll remove $ prefixes for multiple lines as well(in case of `props.language == bash`)